front/src/modules/types/SettingsPath.ts

```diff
@@ -10,4 +10,6 @@ export enum SettingsPath {
   WorkspaceMembersPage = 'workspace-members',
   Workspace = 'workspace',
   Apis = 'apis',
+  NewApi = 'apis/new',
```

Comment:
see comment above, should we rename apis to developers? 
developers/api-key/new
developers/api-key/:id

or api-keys/new and api-keys/:id maybe